application/src/main/java/net/furizon/backend/infrastructure/pretix/service/CachedPretixInformation.java

```diff
@@ -23,23 +32,35 @@
 @RequiredArgsConstructor
 @Slf4j
 public class CachedPretixInformation implements PretixInformation {
-    @NotNull
-    private final UseCaseExecutor useCaseExecutor;
+    @NotNull private final UseCaseExecutor useCaseExecutor;
 
-    @NotNull
-    private final AtomicReference<Event> currentEvent = new AtomicReference<>(null);
+    @NotNull private final AtomicReference<Event> currentEvent = new AtomicReference<>(null);
 
-    @NotNull
-    private final AtomicReference<Integer> questionSecretId = new AtomicReference<>(-1);
+    //Event Struct
 
-    @NotNull
-    private final Cache<Integer, QuestionType> questionTypeIdsCache = Caffeine.newBuilder().build();
+    //Contains tickets, memberships, sponsors, rooms
+    @NotNull private final Cache<CacheItemTypes, Set<Integer>> itemIdsCache = Caffeine.newBuilder().build();
 
-    @NotNull
-    private final Cache<Integer, String> questionIdentifiersCache = Caffeine.newBuilder().build();
+    //Questions
+    @NotNull private final AtomicReference<Integer> questionSecretId = new AtomicReference<>(-1);
+    @NotNull private final Cache<Integer, QuestionType> questionIdToType = Caffeine.newBuilder().build();
+    @NotNull private final Cache<Integer, String> questionIdToIdentifier = Caffeine.newBuilder().build();
+    @NotNull private final Cache<String, Integer> questionIdentifierToId = Caffeine.newBuilder().build();
 
-    @NotNull
-    private final Cache<String, Integer> questionIdentifiersToIdCache = Caffeine.newBuilder().build();
+    //Tickets
+    @NotNull private final Cache<Integer, Integer> dailyIdToDay = Caffeine.newBuilder().build(); //map id -> day idx
+
+    //Sponsors
+    @NotNull private final Cache<Integer, Sponsorship> sponsorshipIdToType = Caffeine.newBuilder().build();
+
+    //Extra days
```

Comment:
why we need comments? the variable should be self-explained, no? ;3

Reply:
So my adhd ass can directly find the correct block to look at hahaha